google/cloud/forseti/services/tracing.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Forseti gRPC tracing setup."""
+
+from opencensus.trace.tracer import Tracer
+from opencensus.trace.samplers import always_on
+from opencensus.trace.exporters import stackdriver_exporter, file_exporter
+from opencensus.trace.exporters.transports import background_thread
+from opencensus.trace.ext.grpc import client_interceptor, server_interceptor
+
+from google.cloud.forseti.common.util import logger
+
+LOGGER = logger.get_logger(__name__)
+
+
+def trace_client_interceptor(endpoint):
```

Comment:
Method naming should generally start with verbs.  So, `create_trace_client_interceptor()`.

Reply:
I'd simplify it to `create_client_interceptor` and `create_server_interceptor` then.

I think it looks good that way:
```
from google.cloud.forseti.services import tracing
tracing.create_client_interceptor(endpoint='localhost:5001')
tracing.create_server_interceptor()
```

When we do the monitoring piece, we'll have similarly:
```
from google.cloud.foresti.services import monitoring
monitoring.create_client_interceptor(endpoint='localhost:5001')
monitoring.create_server_interceptor()
```

Side question: should we move `tracing.py` (and future `monitoring.py`) into `google/cloud/forseti/common/` or does it make sense to keep it next to the `client.py` and `server.py` ?

Reply:
Your naming proposal sounds great!

I am neutral on moving this to the common package, as that to me is for sharing between different packages.  Do you think that will be the case in the future?

Reply:
Yes, I think one of the goals after this PR is to implement some kind of decorator to enable more tracing (sub-spans) on certain functions / classes. In that case those would be shared by other services.